src/font_descriptor.rs

```diff
@@ -250,7 +250,11 @@ impl CTFontDescriptor {
             let value = CFType::wrap_under_create_rule(value);
             assert!(value.instance_of::<CFURL>());
             let url = CFURL::wrap_under_get_rule(mem::transmute(value.as_CFTypeRef()));
-            Some(format!("{:?}", url))
+            let path = CFString::wrap_under_create_rule(CFURLCopyFileSystemPath(
```

Comment:
What if we made this return `Option<PathBuf>` and called `url.to_path()` instead?

Reply:
Oh, deja vu. I understand better now.